api/api/controllers/search_controller.py

```diff
@@ -291,8 +290,8 @@ def build_search_query(
     if not search_params.validated_data["include_sensitive_results"]:
         search_queries["must_not"].append(Q("term", mature=True))
     # Exclude dynamically disabled sources (see Redis cache)
-    if excluded_providers_query := get_excluded_providers_query():
-        search_queries["must_not"].append(excluded_providers_query)
+    if filtered_providers_query := get_filtered_providers_query():
+        search_queries["must"].append(filtered_providers_query)
```

Comment:
I think this should use `filter` instead of `must`. `must` will affect the scoring of the documents, whereas `filter` doesn't. Documentation here about that difference: https://www.elastic.co/guide/en/elasticsearch/reference/8.10/query-dsl-bool-query.html

```suggestion
        search_queries["filter"].append(filtered_providers_query)
```

This might also influence the terminology change I suggested. If we are using "filter" for the ES portion, maybe "filtered providers" is accurate (though it still reads with a negative/exclusion connotation to me, might just be my personal reading not shared by others).

Reply:
Yes, this query should use `filter` instead of `must` because it's more performant, and we don't need the relevancy score for providers. It's a bit unintuitive that the opposite of `must_not` query is the `filter` query, not `must` query.

Reply:
Thanks for letting me know about the `filter` clause. I wasn't aware of it, but reading the documentation, it totally makes sense.